ddtrace/contrib/httplib/patch.py

```diff
@@ -60,9 +59,14 @@ def _wrap_putrequest(func, instance, args, kwargs):
         method, path = args[:2]
         scheme = 'https' if isinstance(instance, httplib.HTTPSConnection) else 'http'
         port = ':{port}'.format(port=instance.port)
+
+        # strip path of non-path data
+        path = parse.urlparse(path).path
+
         if (scheme == 'http' and instance.port == 80) or (scheme == 'https' and instance.port == 443):
             port = ''
         url = '{scheme}://{host}{port}{path}'.format(scheme=scheme, host=instance.host, port=port, path=path)
```

Comment:
Should we use `parse.urlunparse(())` like we did below?

Reply:
this is not a required change, just an observation

Reply:
for now copy-and-pasted code from requests though we should consider down the line this as a good function for metadata sanitization.